src/index.html

```diff
@@ -4,9 +4,259 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+      <img class="logo-img"
+       src="./images/logo.png"
+       alt="logo">
+      <nav class="nav">
+        <a class="nav__link is-active" href="#">Apple</a>
+        <a class="nav__link" href="#">Samsung</a>
+        <a class="nav__link" href="#">Смартфоны и телефоны</a>
+        <a
+         class="nav__link"
+         data-qa="nav-hover"
+         href="#"
+         >
```

Comment:
```suggestion
        <a
         class="nav__link"
         data-qa="nav-hover"
         href="#"
       >
```
Открывающая и закрывающая скобки на одной линии

Reply:
хорошо понял, спасибо